src/data/javascript-quiz.ts

```diff
@@ -1784,6 +1784,18 @@ const javascriptQuiz = [
     Link:
       "https://www.freecodecamp.org/news/event-propagation-event-bubbling-event-catching-beginners-guide/",
   },
+  {
+    Question:
+      "In JavaScript, What is the result if we do a typeof to NaN?",
+    Answer: "Number",
+    Distractor1: "Object",
+    Distractor2: "Number",
```

Comment:
distractor are for wrong answers only

```suggestion
    Distractor2: "String",
```